modules/adblocker/sources/content.es

```diff
@@ -29,6 +30,11 @@ registerContentScript({
       true, /* enable mutation observer */
       async (payload) => {
         const result = await CLIQZ.app.modules.adblocker.action('getCosmeticsFilters', payload);
+        if (USE_PUSH_INJECTIONS_ON_NAVIGATION_EVENTS && result) {
+          // prevent double-injection of scripts (the adblocker is
+          // configured to inject them via tabs.executeScript)
+          result.scripts = [];
+        }
```

Comment:
Could we do that from background instead? This would allow to not include the config (which I believe might be pretty big, not sure if it's already included by other modules) and not transfer data that will not be used.

Reply:
Should be possible, I'll have a look. I agree that it would make more sense to filter it in background.